src/layouts/partials.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { Footer, Header, Main } from './partials';
+
+describe(Header, () => {
+  it('should successfully display the header element', () => {
+    const markup = shallow(<Header location="London" isPlatformAdmin={false} />);
+    expect(markup.find('header nav li a.paas-govuk-tag').text()).toMatch('London');
+    expect(markup.find('header nav li.admin')).toHaveLength(0);
+    // The following is for simply compliance with the design system.
+    // https://github.com/alphagov/govuk-frontend/issues/1688
+    expect(markup.find('header.govuk-header .govuk-header__logotype svg image')
+      .filterWhere(item => item.prop('src') === '/assets/images/govuk-logotype-crown.png')).toHaveLength(1);
+  });
+
+  it('should show the admin link if platform admin', () => {
```

Comment:
An inverse test *should not show the admin link if not platform admin* would be lovely too. 💚

Reply:
I was lazy checking for that in the previous test on line 10, without description. "Two birds with one stone" kind of thing...

I'm happy to separate it into it's own block if it'd make it simpler to read/find? :)